### PR TITLE
BUG: ICON data paths

### PR DESCRIPTION
**Pull request by jklenzing**
Updates ICON data paths to accomodate zip files on FTP server.

Once files are migrated over to SPDF, this will be replaced by the standard cdaweb routines.

### CODE REVIEW

**Comment by jklenzing** [comment]
Tests passing locally.  Need to be merged for #27 to be tested.

File: pysatNASA/instruments/icon_mighti.py [hunk 1]
```diff
@@ -41,11 +41,17 @@
...
----------
+-------
 Originated from EUV support.
 Jeff Klenzing, Mar 17, 2018, Goddard Space Flight Center
 Russell Stoneback, Mar 23, 2018, University of Texas at Dallas
-Conversion to MIGHTI, Oct 8th, 2028, University of Texas at Dallas
+Conversion to MIGHTI, Oct 8th, 2018, University of Texas at Dallas
+
+Note:
+-----
```
Comment (aburrell) [inline pysatNASA/instruments/icon_mighti.py:51]:
```suggestion
Note
----
```

**Review by aburrell — APPROVED** [review]
A small docstring style correction and you're good to go!

**Comment by jklenzing** [comment]
Slight decrease in coverage for cnofs_ivm.  Make sure to check this in the preprocess branch.

**Pull request merged**